coeus-webapp/src/main/webapp/css/common/core.css

```diff
@@ -298,4 +298,23 @@ li.kc-listIconDropdown li a.uif-boxLayoutHorizontalItem  {
 
 li.kc-listIconDropdown li div.uif-boxSection  {
     display: inline-block;
+}
+div.jstree a ins {
+    display:none;
+}
+div.jstree div.uif-field span a {
+    display:inline-block;
+    line-height:16px;
+    height:16px;
+    color: #07c;
+    white-space:nowrap;
+    text-decoration:none;
+    padding:1px 2px;
+    margin:0;
+}
+div.jstree li a.IP:hover, div.jstree li a.DP:hover,
+div.jstree li a.irb:hover,div.jstree li a.iacuc:hover,
+div.jstree li a.neg:hover,div.jstree li a.award:hover,
+div.jstree li a.subaward:hover{
```

Comment:
Is there a reason we have all these classes? Can't we just use a global a.medusa or something? Also, the jstree usage here is a little concerning. I feel like we might use jstree outside of KRAD and outside of medusa. Is this styling intended to be global across the app? If not, maybe add a class to the jstree for medusa use that instead?